requirements.txt

```diff
@@ -5,4 +5,4 @@ mock==2.0.0
 paho-mqtt==1.2
 pint==0.7.2
 websocket-client==0.37.0
-
+uptime==3.0.1
```

Comment:
There is one more requirements.txt for liota-lite. You will need to modify that too.

Reply:
yes